### PR TITLE
🌐 Add French translation for `docs/fr/docs/advanced/path-operation-advanced-configuration.md`

### PR DESCRIPTION
**Pull request by axel584**
I close my old PR : https://github.com/tiangolo/fastapi/pull/5701
I screwed up somewhere and this old PR contained 109 modified files, I preferred to start on a new one by taking the commits with a cherry-pick

### CODE REVIEW

**Comment by axel584** [comment]
@rjNemo / @frabc / @JulianMaurin / @emmrichard / @raphaelauv : if you can approve it, I'll thank you :pray:

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+!!! note "Détails techniques"
+    La spécification OpenAPI appelle ces métaonnées des <a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#operation-object" class="external-link" target="_blank">Objets d'opération</a>.
+
+Il contient toutes les informations sur le *chemin* et est utilisé pour générer automatiquement la documentation.
+
+Il inclut les `tags`, `parameters`, `requestBody`, `responses`, etc.
+
+Ce schéma OpenAPI spécifique aux *operations* est normalement généré automatiquement par **FastAPI**, mais vous pouvez également l'étendre.
+
+!!! tip
```
Comment (frabc) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:77]:
```suggestion
!!! Astuce
```

**Review by cfraboulet — COMMENTED** [review]
@axel584 After the change I mentioned, it's good for me. I will validate your PR :)

**Review by cfraboulet — APPROVED** [review]
LGTM 😊 Well done 🚀

**Review by emmrichard — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @axel584! :cake: 

And thanks for the reviews @emmrichard and @frabc :coffee:

**Pull request merged**